Src/BootCamp.Chapter/ConsolePrompter.cs

```diff
@@ -0,0 +1,80 @@
+﻿using BootCamp.Chapter.Logging;
+using System;
+
+namespace BootCamp.Chapter
+{
+    public static class ConsolePrompter
+    {
+        public static ILogger SetupSession()
+        {
+            Console.Write("Please select where would you like to log for this session. 0 = Console, 1 = Logfile: ");
+
```

Comment:
Not sure if I did right by calling the LoggerFactories inside of ConsolePrompter, but I did not want to place this huge block of code inside of HealthCheckSimulator.